src/rebar_file_utils.erl

```diff
@@ -189,16 +189,23 @@ cp_r([], _Dest) ->
     ok;
 cp_r(Sources, Dest) ->
     case os:type() of
-        {unix, _} ->
+        {unix, Os} ->
             EscSources = [rebar_utils:escape_chars(Src) || Src <- Sources],
             SourceStr = rebar_string:join(EscSources, " "),
+            Source = case Os of
+                darwin ->
+                    string:sub_string(SourceStr, 1, length(SourceStr) - 1);
```

Comment:
I don't think you can assume that the path always contains a /

Reply:
It is, I see. Will have to use regex I suppose